perspective_reddit_bot/check_mod_actions.py

```diff
@@ -43,6 +43,7 @@ def write_moderator_actions(reddit,
   else:
     record['approved'] = None
     record['removed'] = None
+  record['action_checked_utc'] = datetime.utcnow().strftime('%Y%m%d_%H%M%S')}
```

Comment:
I feel like this function/method combination is used frequently enough (2-4 times?) to warrant being pulled into a little one-line function.

Reply:
Good point. Will do in a future PR with some other minor refactorings.